themes/inbeat/assets/js/main.js

```diff
@@ -163,6 +163,35 @@ function testimonial() {
 }
 
 function pricing() {
+    //set toggle 
+    var toggle = document.getElementsByClassName('toggle-container')
+    if(toggle.length === 0){return}
+    var currentActive = toggle[0];
+    var creatorStudioActive = function (item) {
+        item.classList.add('--active')
+        document.querySelector('.toggle_database').classList.remove('--active')
+        document.querySelector('.pricing_creator-studio').classList.add('--show')
+        document.querySelector('.pricing_database').classList.remove('--show')
+    };
```

Comment:
why this definition needs to be inside a function? is it possible to take it out of `pricing` function?

Reply:
This is intentionally placed inside pricing function, since it has no scope outside `pricing` method, Also similar pattern was used in other methods in this file.